### DISCUSSION TITLE
Retweet Puanları

### DISCUSSION THREAD

**Discussion by ridvanbesli** [IBAX-io/go-ibax#1833] [Ideas]
Görev listesinde yer alan son üç etkinliğin puanları tanımlanmadı. Bu tweetleri retweetleyen kişileri sistemden manuel olarak tespit edip puanları verilebilir mi?

**Comment by Abdullahoda** [comment]
Son 3 retweet puanlarını geri alıcaklar mış haksızlık olmasın diye mesajıma cevaplandı dersen sevinirim kardeşim
